tests/phpunit/elementor/modules/atomic-widgets/style-transformers/test-linked-dimensions-transformer.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Elementor\Testing\Modules\AtomicWidgets\PropTypes;
+
+use Elementor\Modules\AtomicWidgets\Styles\Transformers\Array_Transformer;
+use Elementor\Modules\AtomicWidgets\Styles\Transformers\Linked_Dimensions_Transformer;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Test_Linked_Dimensions_Transformer extends Elementor_Test_Base {
+
+	public function test_transform__returns_linked_dimensions() {
+		// Arrange.
+		$link_dimensions_transformer = new Linked_Dimensions_Transformer();
+
+		// Act.
+		$transformed_value = $link_dimensions_transformer->transform( [
+			'isLinked' => true,
+			'top' => '10px',
+			'right' => '20px',
+			'bottom' => '30px',
+			'left' => '40px',
+		], function( $value ) {
+			return $value;
+		} );
+
+		// Assert.
+		$this->assertSame( '10px 20px 30px 40px', $transformed_value );
```

Comment:
If `isLinked === true` shouldn't all values be the same?
Though in theory I assume that if the "link" button is clicked - all values should be changes automatically from the UI, but maybe even if not but the value is true - enforce one of the values to be used for all edges?

Reply:
It is just a UI implementation, the linked property is meaningless to the transformer. The only reason it exists is that we need to preserve the state of the UI.